molgenis-metadata-manager/src/main/frontend/src/store/getters.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Returns a property object used for creating a sweetalert2 modal when trying to delete an EntityType
+ */
+export const getConfirmBeforeDeletingProperties = (identifier) => {
+  return {
+    title: 'You are about to delete ' + identifier,
+    text: 'Are you sure you want to continue?',
+    type: 'warning',
+    showCancelButton: true,
+    confirmButtonText: 'Yes, delete',
+    cancelButtonText: 'Cancel'
+  }
+}
+
+/**
+ * Returns a property object used for creating a sweetalert2 modal when leaving a view with unsaved changes
+ */
+export const getConfirmBeforeLeavingProperties = () => {
+  return {
+    title: 'There are unsaved changes',
+    text: 'All unsaved changes will be lost, are you sure you want to continue?',
+    type: 'warning',
+    showCancelButton: true,
+    confirmButtonText: 'Yes',
+    cancelButtonText: 'No, stay'
+  }
+}
+
+export default {
+  /**
+   * Returns the EntityType object based on the ID of the selected EntityType
+   */
+  getSelectedEntityType: state => state.entityTypes.find(entityType => entityType.id === state.selectedEntityTypeId),
+  /**
+   * Returns the Attribute object based on the ID of the selected Attribute
+   */
+  getSelectedAttribute: state => state.editorEntityType && state.editorEntityType.attributes.find(attribute => attribute.id === state.selectedAttributeId),
```

Comment:
Case of getting used to lambda's I think,
else it would be
```js
getSelectedAttribute: function (state) {
    if (state.editorEntityType) {
       return  state.editorEntityType.attributes.find(function (attribute) {
            return attribute.id === state.selectedAttributeId
        }
    } else {
        return null
    }
}
```